packages/@aws-cdk/aws-cloudtrail/test/test.cloudtrail.ts

```diff
@@ -160,10 +160,12 @@ export = {
     'with cloud watch logs': {
       'enabled'(test: Test) {
         const stack = getTestStack();
-        new Trail(stack, 'MyAmazingCloudTrail', {
+        const t = new Trail(stack, 'MyAmazingCloudTrail', {
           sendToCloudWatchLogs: true,
         });
 
+        test.ok(t.logGroup);
+        test.deepEqual(stack.resolve(t.logGroup!.logGroupArn), { 'Fn::GetAtt': ['MyAmazingCloudTrailLogGroup2BE67F87', 'Arn'] });
```

Comment:
how come `!.` - @eladb had mentioned in another PR that we generally try to avoid it in our codebase. Although this is a test, I think we should try and be consistent if it's a style we generally don't prefer

Reply:
This test verifies that `t.logGroup` is not `undefined` and returns an ARN (vs. the other case where it returns `undefined`).

How should I update my assertions?

Reply:
unfortunately, something a little more naive - here's the similar [example](https://github.com/aws/aws-cdk/pull/8022#discussion_r427846927) in the PR i mentioned

Reply:
Sorry, I'm not following what the change you're expecting here is. The unit testing in the linked PR is testing something quite different from what I'm trying to here.

Can you tell me what's wrong with the current assertion, and how you would write it?
I can either assert for it being not `undefined` or for the value it returns. I've chosen the latter. Is there a 3rd option?

Reply:
I'd do the former, but I'm not picky on this. go for it!